plugins/catalog-backend-module-azure/src/lib/azure.ts

```diff
@@ -47,7 +50,7 @@ export async function codeSearch(
   const searchBaseUrl = isCloud(azureConfig.host)
     ? 'https://almsearch.dev.azure.com'
     : `https://${azureConfig.host}`;
-  const searchUrl = `${searchBaseUrl}/${org}/${project}/_apis/search/codesearchresults?api-version=6.0-preview.1`;
+  const searchUrl = `${searchBaseUrl}/${org}/_apis/search/codesearchresults?api-version=6.0-preview.1`;
```

Comment:
Does it make sense to update the api-version to `api-version=7.0`?

Reply:
No, Azure DevOps Server on-prem doesn't support that version yet I've also already tested this.